src/App.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './App.css';
+import Calculator from './components/calculator';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <Calculator />
+      </div>
+    );
+  }
+}
+
```

Comment:
- [ ] As I mentioned in my above comment you should use functional component not class-based component so please follow the instructions and use the functional components because the functional components approach makes our code more maintainable, easy to read and even reduce the number of lines of code. On the other hand side, the new hooks are only accessible in the function-based component, So sooner or later you would switch to a functional styling approach and I think it is the best time to do. 

The best part about the `functional` based component approach is it makes our life easier to implement the side effects in a more efficient way, about which you will learn new things soon in upcoming milestones. Below is my suggestion and you can compare the both styles to visualize the beauty of the function based component.

```suggestion
function App () {
    return (
      <div className="App">
        <Calculator />
      </div>
    );
  }
```

Reply:
Noted will work on it.